src/index.css

```diff
@@ -0,0 +1,68 @@
+@font-face {
```

Comment:
I would suggest you try and implement  SASS/SCSS in this project (for the sake of learning) instead of plain CSS

Reply:
Changed